src/ccid_usb.c

```diff
@@ -154,6 +154,7 @@ static struct _bogus_firmware Bogus_firmwares[] = {
 	{ 0x03F0, 0x0036, 0x0124 }, /* HP USB CCID Smartcard Keyboard */
 	{ 0x062D, 0x0001, 0x0102 }, /* THRC Smart Card Reader */
 	{ 0x04E6, 0x5291, 0x0112 }, /* SCM SCL010 Contactless Reader */
+	{ 0x076B, 0x3A21, 0x0103 }, /* HID Smart@Link generic */
```

Comment:
why do you add the reader in the Bogus_firmwares[] list?

The reader firmware is already at version 1.03 so no warning will be displayed.